### DISCUSSION TITLE
Red Hat Linux (RHL) support

### DISCUSSION THREAD

**Discussion by varonix0** [yao-pkg/pkg#81] [Q&A]
Is RHL currently supported, and if not, are there any plans to support it? It's a major Linux distro that's very widely used by enterprises around the globe.

Thanks!

**Comment by robertsLando** [comment]
We support linux so yeah I think it should work :)

**Reply by varonix0** [reply]
Thanks @robertsLando! I'll test this out shortly and let you know :)

**Reply by varonix0** [reply]
@robertsLando It doesn't seem to be possible to pass the `--targets` flag to target RHL 🤔

**Reply by DanielRuf** [reply]
Does the normal flag for Linux not work for you?
